user/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+
+class User(models.Model) :
+	username          = models.CharField(max_length = 20)
+	sex	              = models.CharField(max_length = 10)
+	email	          = models.CharField(max_length = 50)
```

Comment:
```suggestion
	email	          = models.CharField(max_length = 50, unique=True)
```
실제로 고유해야하는 데이터는 유일성을 지켜주세요.